site.js

```diff
@@ -454,7 +455,8 @@ export default class Site {
         ...this.extraData,
       };
 
-      content = await engine.render(layoutData.content, pageData, path);
+      content = await engine.render(layoutData.content, pageData, page.src.path + page.src.ext);
```

Comment:
Here you must use the `path` variable, instead `page.src.path + page.src.ext`. `path` is the path of the layout but you're passing the path of the page that uses this layout.